plugins/headless/docs/headless-gamma-webserver.md

```diff
@@ -21,7 +21,7 @@ This section details the API used to communicatie with the webserver.
   - **POST**
 	- **addProject** `/gamma/workspace/{workspace}/project` - Adds a project to the specified workspace. The request body is a multipart/form-data type body, which has a `file` field, where the zip file containing the project must be provided (examples of this can be found in the _Example: Workflow with Docker_ section). (Alternatively, in Postman, the file can be selected from the file system). The zip file must have the same name as the project. As an example, the `hu.bme.mit.gamma.test` project should be placed in `hu.bme.mit.gamma.test.zip`.
 	- **getResult** `/gamma/workspace/{workspace}/project/{project}` - Gets specified files and folders from the project found in the given workspace and zips them. The files and/or folders should be specified as raw data in the request body, specifying the relative paths from the project root, as `files`. Example: `"files":["src-gen","test-gen","trace/ExecutionTrace0.get"]`. If the given file or folder does not exists, it will not appear in the zip file (the request doesn't throw an exception).
-	- **addWorkspace** `/gamma/workspace` - Creates a workspace in the location specified in `config.properties`. Returns with the name (ID) of the workspace, which is used in further requests.
+	- **addWorkspace** `/gamma/workspace` - Creates a workspace in `hu.bme.mit.gamma.headless.server/workspaces`. Returns with the name (ID) of the workspace, which is used in further requests.
```

Comment:
Well, we still rely on the the location specified in `config.properties`, don't we? Yes, the default location is set to `hu.bme.mit.gamma.headless.server/workspaces`, but this can be overwritten by the user if they find it necessary. So keep the original sentence and extend it with the currently specified (default) path inside parentheses: "Creates a workspace in the location specified in `config.properties` (set to `hu.bme.mit.gamma.headless.server/workspaces` by default)."